src/components/Molecules/ArticleTeaser/ArticleTeaser.js

```diff
@@ -31,36 +33,48 @@ const Link = styled(link)`
   }
 
   @media ${({ theme }) => theme.breakpoint('medium')} {
-    flex-direction: ${({ category }) => (category ? 'row' : 'column')};
+    flex-direction: ${({ category }) => !category && 'column'};
   }
 `;
 
 const ImageWrapper = styled.div`
-  height: auto;
-  @media ${({ theme }) => theme.breakpoint('small')} {
-    width: ${({ category }) => (category ? '70px' : '45%')};
-  }
+  padding-left: ${({ category }) => category && `${spacing('md')}`};
+  display: flex;
+  align-items: center;
+  justify-content: center;
 
-  @media ${({ theme }) => theme.breakpoint('medium')} {
-    width: ${({ category }) => (category ? '70px' : '100%')};
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    flex: ${({ category }) => !category && '0 0 45%'};
   }
 `;
 
 const CopyWrapper = styled.div`
   padding: ${spacing('l')};
-  @media ${({ theme }) => theme.breakpoint('small')} {
-    width: ${({ category }) => (category ? '100%' : '55%')};
-  }
-
-  @media ${({ theme }) => theme.breakpoint('medium')} {
-    width: 100%;
-  }
+  word-break: break-all;
```

Comment:
We don't wanna be breaking-up words like this:

![Uploading Screen Shot 2019-09-13 at 12.03.43.png…]()


Could we remove that rule? Looks like the text is already set-up to break properly

Reply:
Sure, done